src/app/Rules/Rules.tsx

```diff
@@ -96,7 +97,15 @@ export const Rules = () => {
       title: 'Archival Period',
       transforms: [
         info({
-          tooltip: 'Period in seconds. Cryostat will connect to matching targets at this interval and copy the relevant recording data into its archives. Values less than 1 prevent data from being copied into archives - recordings will be started and remain only in target JVM memory.'
+          tooltip: 'Period in seconds. Cryostat will connect to matching targets at this interval and copy the relevant recording data into its archives. Values less than 1 prevent data from being repeatedly copied into archives - recordings will be started and remain only in target JVM memory.'
+        })
+      ],
+    },
+    {
+      title: 'Initial Delay',
+      transforms: [
+        info({
+          tooltip: 'Initial delay in seconds. Cryostat will wait this amount of time before first copying recording data into its archives. Values less than 0 default to equal to the Archival Period. You can set a non-zero Initial Delay with a zero Archival Period, which will start a recording and copy it into archives exactly once after a set delay.'
```

Comment:
> Values less than 0 default to equal to the Archival Period.

The description is for the last paragraph [above](https://github.com/cryostatio/cryostat-web/pull/476/#issue-1290602429) right? This might be mistaken as:  `initialDelaySeconds` will be set equal to the value `ArchivePeriod` if left as 0?

I am thinking maybe it could be like `Values less than 0 has no effect?` or maybe another better suggestion.

Reply:
> This might be mistaken as: initialDelaySeconds will be set equal to the value ArchivePeriod if left as 0?

[That's correct, though.](https://github.com/cryostatio/cryostat/pull/1015/files#diff-35494c48ebb256b9fb6036f153781a729d47eaa5e68cc8ed6b65664b164f16c6R78) If the backend really treated `initialDelaySeconds: 0` at face value, then that should indicate that an archived copy will be made at time `x+0`, but the backend specifically avoids that case.

Reply:
Ahhh yesss that makes sense! Maybe I got a bit mixed up there :D